components/director/internal/domain/api/fixtures_test.go

```diff
@@ -0,0 +1,344 @@
+				Condition: graphql.FetchRequestStatusConditionInitial,
+				Timestamp: graphql.Timestamp(time),
+			},
+		},
+	}
+
+	deprecated := false
+	deprecatedSince := ""
+	forRemoval := false
+
+	version := &graphql.Version{
+		Value:           "1.0.0",
+		Deprecated:      &deprecated,
+		DeprecatedSince: &deprecatedSince,
+		ForRemoval:      &forRemoval,
+	}
+
+	headers1 := graphql.HttpHeaders{"testHeader": {"hval1", "hval2"}}
+	headers2 := graphql.HttpHeaders{"testHeader": {"hval1", "hval2"}}
+	params1 := graphql.QueryParams{"testParam": {"pval1", "pval2"}}
+	params2 := graphql.QueryParams{"testParam": {"pval1", "pval2"}}
+
+	auth1 := graphql.Auth{
+		Credential: &graphql.BasicCredentialData{
+			Username: "test",
+			Password: "pwd",
+		},
+		AdditionalHeaders:     &headers1,
+		AdditionalQueryParams: &params1,
+		RequestAuth:           &graphql.CredentialRequestAuth{},
+	}
+	auth2 := graphql.Auth{
+		Credential: &graphql.BasicCredentialData{
+			Username: "test",
+			Password: "pwd",
+		},
+		AdditionalHeaders:     &headers2,
+		AdditionalQueryParams: &params2,
+		RequestAuth:           &graphql.CredentialRequestAuth{},
+	}
+
+	runtimeAuth1 := graphql.RuntimeAuth{
+		RuntimeID: "1",
+		Auth:      &auth1,
+	}
+	runtimeAuth2 := graphql.RuntimeAuth{
+		RuntimeID: "2",
+		Auth:      &auth2,
+	}
+
+	return &graphql.APIDefinition{
+		ID:            id,
+		ApplicationID: "applicationID",
+		Name:          name,
+		Description:   &description,
+		Spec:          spec,
+		TargetURL:     "https://test-url.com",
+		Group:         &group,
+		Auth:          &runtimeAuth1,                                        //TODO: https://github.com/kyma-incubator/compass/issues/67
+		Auths:         []*graphql.RuntimeAuth{&runtimeAuth1, &runtimeAuth2}, //TODO: https://github.com/kyma-incubator/compass/issues/67
+		DefaultAuth:   &auth1,                                               //TODO: https://github.com/kyma-incubator/compass/issues/67
+		Version:       version,
+	}
+}
+
+func fixModelApiDefinitionInput(name, description string, group string) *model.APIDefinitionInput {
+
+	data := []byte("data")
+	mode := model.FetchModeSingle
+	format := model.SpecFormatYaml
+	filter := "filter"
+
+	spec := &model.APISpecInput{
+		Data:   &data,
+		Type:   model.APISpecTypeOpenAPI,
+		Format: &format,
+		FetchRequest: &model.FetchRequestInput{
+			URL:    "https://test-fetch-request.com",
+			Auth:   nil, //TODO: https://github.com/kyma-incubator/compass/issues/67
+			Mode:   &mode,
+			Filter: &filter,
+		},
+	}
+
+	deprecated := false
+	deprecatedSince := ""
+	forRemoval := false
+
+	version := &model.VersionInput{
+		Value:           "1.0.0",
+		Deprecated:      &deprecated,
+		DeprecatedSince: &deprecatedSince,
+		ForRemoval:      &forRemoval,
+	}
+
+	credentialRequestAuthInput := model.CredentialRequestAuthInput{}
+	basicCredentialDataInput := model.BasicCredentialDataInput{
+		Username: "test",
+		Password: "pwd",
+	}
+	authInput := model.AuthInput{
```

Comment:
BTW Don't pass all values for Auth, as the Auth converter is tested somewhere else. What you need to do is to test only API converter and that's it. You're mocking Auth converter anyway, so please decrease complexity here.